### PR TITLE
Add assisted capture to ZividBenchmark

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This adds the training phase of capture assistant (suggest settings).

### CODE REVIEW

**Comment by SatjaSivcev** [comment]
Please review @torbsorb @knzivid

File: source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp [hunk 1]
```diff
@@ -317,6 +327,36 @@ namespace
...
+    {
+        printAssistedCapture3DHeader(numFrames);
+
+        Zivid::CaptureAssistant::SuggestSettingsParameters suggestSettingsParameters(
+            std::chrono::milliseconds{ 1200 }, Zivid::CaptureAssistant::AmbientLightFrequency::none);
+
+        for(size_t i = 0; i < 5; i++) // Warmup
+        {
+            const auto settingsVector{ Zivid::CaptureAssistant::suggestSettings(camera, suggestSettingsParameters) };
+        }
```
Comment (torbsorb) [inline source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp:340]: is it required to run warmup on this specifically? If so, why is it sufficient?

**Comment by knzivid** [comment]
I do not have any comments on this PR, the changes look good to me

File: source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp (hunk 1 above)
Comment (SatjaSivcev) [reply source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp:340]: The first capture is always much slower here, see the image below. The second capture can be slightly slower (e.g. 880 for 820 average), but this one I saw only once in 15-ish tests.

The observation is similar for 3D captures, but for them, occasionally, a number of first 5 frames can be outliers also (this happens more commonly than above).

I wanted to keep this warmup consistent with the ones in other functions where we use x5, benchmarkCapture2D() is the most similar case to this one and it also uses 5.

Alternatively, we could use 1 or 2 warmup captures for 2D captures and for this. What do you think?

image

**Pull request merged**